Tasks/Gradle/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -24,9 +24,9 @@
   "loc.input.label.codeCoverageTool": "CodeCoverage Tool",
   "loc.input.help.codeCoverageTool": "Select the code coverage tool.",
   "loc.input.label.classFilesDirectories": "Class Files Directories",
-  "loc.input.help.classFilesDirectories": "Comma separated list of directories containing class files, archive files(jar, war etc.). Directories and archives are searched recursively for class files. Code coverage is reported for class files present in the directories. For example, classes1/Tests,classes2/Tests2. Refer https://docs.gradle.org/current/userguide/jacoco_plugin.html.",
```

Comment:
Is this not relative to gradlew?
If it is .. my suggestion is Comma seperated list of relative paths from Gradle Wrapper Script to directories containing class files